front/app/api/authentication/confirm_email/confirmEmail.ts

```diff
@@ -19,12 +20,10 @@ export default async function confirmEmail(
     await streams.add(confirmationApiEndpoint, bodyData);
 
     queryClient.invalidateQueries({ queryKey: requirementsKeys.all() });
+    queryClient.invalidateQueries({ queryKey: meKeys.all() });
```

Comment:
The difference between invalidating and reseting the cache is that in the first place we keep the previous data during the refetch. In many (most!) cases, this is desirable behaviour, the only time we really want to hard reset the me query, as far as I could identify, is on logout. Otherwise, just invalidating is ok.

Reply:
Ahh ok! Yes I misread the code :) Makes perfect sense. thanks!